writers_rollingfilewriter.go

```diff
@@ -449,7 +461,12 @@ func (rw *rollingFileWriter) Write(bytes []byte) (n int, err error) {
 	// needs to roll if:
 	//   * file roller max file size exceeded OR
 	//   * time roller interval passed
-	nr, err := rw.self.needsToRoll()
+	fi, err := os.Stat(rw.rollingTrackerPath())
+	if err != nil {
+		return 0, err
+	}
+	lastRollTime := fi.ModTime()
```

Comment:
Assumption seems to be that last time file was modified was when it was "rolled" -- is this accurate? Would expect last mod time was last time log event was written